projects/tcpdump/load_sydr_packet.c

```diff
@@ -0,0 +1,91 @@
+#include <pcap/pcap.h>
+#include <netdissect-stdinc.h>
+#include <netdissect.h>
+#include <print.h>
+
+#include <errno.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+static uint64_t packets_captured = 0;
+
+void print_packet(unsigned char *user, const struct pcap_pkthdr *h, const unsigned char *sp) {
+  ++packets_captured;
+  pretty_print_packet((netdissect_options *)user, h, sp, packets_captured);
+}
+
+FILE *bufferToFile(const uint8_t *Data, size_t Size) {
```

Comment:
We should directly read from file in sydr target

Reply:
done